php/src/Data/ConfigurationManager.php

```diff
@@ -297,6 +281,29 @@ public function WriteConfig(array $config) : void {
         file_put_contents(DataConst::GetConfigFile(), json_encode($config));
     }
 
+    private function GetEnvironmentalVariableOrConfig(string $envVariableName, string $configName, string $defaultValue) : string {
+        $envVariableOutput = getenv($envVariableName);
+        if ($envVariableOutput === false) {
+            $config = $this->GetConfig();
+            if (!isset($config[$configName]) || $config[$configName] === '') {
+                $config[$configName] = $defaultValue;
+            }
+            return $config[$configName];
+        } else {
```

Comment:
It's a code style preference but since you have a return in the if block, you don't need a block for the else statement.

Reply:
right, will do like suggested. thanks for the review :)